### PR TITLE
8253543: sanity/client/SwingSet/src/ButtonDemoScreenshotTest.java failed with "AssertionError: All pixels are not black"

### PR DESCRIPTION
**Pull request by shurymury**
Added a few synchronization points to the test:
	1	wait for the screen to change where the test window is displayed
	2	wait for the image to get still (i.e. painted complete)
	3	wait for the image to change after the mouse move
	4	wait for the image change and then to get still after the mouse press
	5	wait for the image to change after the mouse release

@mrserb Can you take a look?

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8253543](https://bugs.openjdk.java.net/browse/JDK-8253543): sanity/client/SwingSet/src/ButtonDemoScreenshotTest.java failed with "AssertionError: All pixels are not black"

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/389/head:pull/389`
`$ git checkout pull/389`

### CODE REVIEW

File: test/jdk/sanity/client/SwingSet/src/ButtonDemoTest.java [hunk 1]
```diff
@@ -33,7 +35,7 @@
 import static org.testng.AssertJUnit.*;
 import org.testng.annotations.Test;
 import org.jemmy2ext.JemmyExt.ByToolTipChooser;
-import static org.jemmy2ext.JemmyExt.EXACT_STRING_COMPARATOR;
+import static org.jemmy2ext.JemmyExt.*;
```
Comment (mrserb) [inline test/jdk/sanity/client/SwingSet/src/ButtonDemoTest.java:38]: Looks like the change in this file is unnecessary? Note, we do not use '*' in the imports(same note in the EditorPaneDemoTest.java )

Comment (shurymury) [reply test/jdk/sanity/client/SwingSet/src/ButtonDemoTest.java]: reverted the change of that file

**Review by mrserb — APPROVED** [review]

**Review by amresh-sahu — COMMENTED** [review]
All changes look good.

**Pull request closed without merging**